batch/batch/worker/worker.py

```diff
@@ -1177,9 +1177,11 @@ def _mounts(self, uid, gid):
         external_volumes = []
         volumes = self.image.image_config['Config']['Volumes']
         if volumes:
-            for v_container_path in volumes:
-                if not v_container_path.startswith('/'):
-                    v_container_path = '/' + v_container_path
+            for _v_container_path in volumes:
+                if not _v_container_path.startswith('/'):
+                    v_container_path = '/' + _v_container_path
+                else:
+                    v_container_path = _v_container_path
```

Comment:
This feels like a bug, no? the `destination` that we set in the container should be as specified by the image manifest, not altered to prepend a `/`. This prepending a slash is only for putting it somewhere easy to find on the host.

I think I only added `VOLUME` support to get buildkit working which I presume uses an absolute path which is why we never noticed it, but I presume there's nothing wrong with specifying a `VOLUME` that is relative to the `WORKDIR`.